src/main/java/de/retest/recheck/ignore/RecheckIgnoreUtil.java

```diff
@@ -45,15 +45,18 @@ public static GlobalIgnoreApplier loadRecheckIgnore() {
 		return GlobalIgnoreApplier.create( NopCounter.getInstance() );
 	}
 
-	public static GlobalIgnoreApplier loadRecheckSuiteIgnore( final File ignoreFilesBasePath ) {
+	/**
+	 * Loads a filter from the argument base path, e.g., test suite and step filters.
+	 */
+	public static GlobalIgnoreApplier loadRecheckIgnore( final File ignoreFilesBasePath ) {
 		try {
 			final LoadFilterWorker loadFilterWorker =
 					new LoadFilterWorker( NopCounter.getInstance(), ignoreFilesBasePath );
 			return loadFilterWorker.load();
 		} catch ( final FileNotFoundException e ) {
-			logger.debug( "Ignoring missing suite ignore file." );
+			logger.debug( String.format( "Ignoring missing suite or step ignore file in '%s'", ignoreFilesBasePath ) );
```

Comment:
SLF4J can handle this (see https://www.slf4j.org/faq.html#logging_performance):

```suggestion
			logger.debug( "Ignoring missing suite or step ignore file in '{}'.", ignoreFilesBasePath );
```